src/renderer/assets/themes/material-dark.theme.css

```diff
@@ -23,23 +23,48 @@
   --editorColor04: rgba(171, 178, 191, .04);
   --editorBgColor: #34393f;
   --deleteColor: #ff6969;
-  --iconColor: rgba(255, 255, 255, .8);
+  --iconColor: rgba(255, 255, 255, .56);
   --codeBgColor: #d8d8d869;
   --codeBlockBgColor: #3f454c;
   --inputBgColor: rgba(0, 0, 0, .1);
+  --focusColor: var(--themeColor);
+
+  --buttonPadding: 7px 14px;
+  --buttonBorderRadius: 5px;
+  --buttonFontColor: rgba(255, 255, 255, .6);
   --buttonBgColor: #58606a;
-  --buttonBorderColor: rgba(0, 0, 0, 0.2);
-  --buttonShadow: rgba(0, 0, 0, 0.12);
-  --buttonHover: #4a5058;
-  --buttonActive: #434a53;
+  --buttonBorder: 1px solid #383d44;
+  --buttonShadow: none;
+  --buttonFontColorHover: var(--buttonFontColor);
+  --buttonBgColorHover: #646d78;
+  --buttonBorderHover: var(--buttonBorder);
+  --buttonFontColorActive: var(--buttonFontColor);
+  --buttonBgColorActive: #4a5159;
+  --buttonBorderActive: var(--buttonBorder);
+
+  --buttonPrimaryFontColor: #ffffff;
+  --buttonPrimaryBgColor: var(--themeColor);
+  --buttonPrimaryBorder: none;
+  --buttonPrimaryShadow: none;
+  --buttonPrimaryFontColorHover: var(--buttonPrimaryFontColor);
```

Comment:
With so many CSS variables, can we not reduce it? Because the fewer variable names, the more convenient it is for users to customize the theme in the future. Too many variables can be confusing for users.

Reply:
I think it would be easier to write themes. E.g. the one-dark theme used a lot of overwrites that need to be updated each time we change a bit of GUI. With variables it's much easier to maintain a theme without breaking it after an update.

Reply:
I still think that too many variables are not good for maintenance, nor is it a best practice. If you want to reuse styles, we can define some common classes, such as `btn`, `size-small`, `primary`, `error`, etc. instead of defining so many variables. name.